src/main/webapp/characterDesign.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Character Design</title>
+    <link rel="stylesheet" href="characterDesign.css">
+    <script src="characterDesign.js"></script>
+  </head>
+  <body onload="fetchBlobstoreUrl()">
+    <h1>Character Design</h1>
+    <p>Type a nickname and click submit:</p>
+
+    <form id="my-form" class="hidden" method="POST" enctype="multipart/form-data">
+      <p>Type nickname:</p>
+      <input name="message" required></input>
+      <br/>
+      <p>Upload an image:</p>
+      <input type="file" name="image" required>
```

Comment:
Should the image be required or should there maybe be a default if none is uploaded?